src/mame/layout/irrmaze.lay

```diff
@@ -46,41 +51,43 @@ copyright-holders:Vas Crabb
 	</element>
 
 	<element name="trackball">
-		<disk state="0"><color red="0.6" green="0" blue="0" /></disk>
-		<disk state="1"><color red="0.8" green="0.7" blue="0.1" /></disk>
+		<disk state="0"><color red="0.5" green="0" blue="0" /></disk>
+		<disk state="1"><color red="1" green="0.1" blue="0.25" /></disk>
```

Comment:
Can you provide a reference for this?  Every video I've seen of the cabinet shows the trackball being a lot more yellow when illuminated by the incandescent globe.  You've made it look decidedly pink which is definitely not going to happen with incandescent illumination.

Reply:
I've used this [video](https://www.youtube.com/watch?v=a7Rhc68Mg84) and [photo](https://pbs.twimg.com/media/CCecrXnUMAEQ87G.jpg:large) as a reference for the pink shade. The original trackball is red as shown in the [kit](https://www.neostore.com/The-Irritating-Maze-English-MVS-kit-p/1601.htm) and [flyer](https://flyers.arcade-museum.com/flyers_video/snk/14018501.jpg). When lit up, it will turn to a lighter shade of red, which will probably be interpretet as either bright orange or pink depending on the camera.